### PR TITLE
Update required media item fields

### PR DESCRIPTION
**Pull request by mdholloway**
As of https://gerrit.wikimedia.org/r/#/c/436360/ (adding support for
Mathoid images) media items may not have the usual set of Commons
metadata.  This updates the media spec to fix the monitoring check.

### CODE REVIEW

**Comment by Pchelolo** [comment]
Congrats @mdholloway - you've made PR #1000 :)

**Comment by mdholloway** [comment]
\o/

**Comment by Pchelolo** [comment]
1 test is still failing for the media endpoint.

**Comment by d00rman** [comment]
@mdholloway It would also be good for RB's and MCS' specs to have the same `x-maples` for the same end points, so that when things start failing we can identify the source more easily. Right now RB is alerting because of the change in MCS, but MCS is not.

**Comment by Pchelolo** [comment]
@mdholloway test is still failing with `body item original not found in the response`

**Comment by mdholloway** [comment]
> @mdholloway It would also be good for RB's and MCS' specs to have the same x-maples for the same end points, so that when things start failing we can identify the source more easily. Right now RB is alerting because of the change in MCS, but MCS is not.

See https://gerrit.wikimedia.org/r/#/c/437636/.  Small change to follow in restbase as well.

File: v1/media.yaml [hunk 1]
```diff
@@ -254,11 +245,9 @@ definitions:
         type: string
         description: description of the image from Wikimedia Commons
     required:
-      - titles
-      - file_page
+      - section_id
       - type
-      - thumbnail
-      - license
+      - original
```
Comment (d00rman) [inline v1/media.yaml]: There is something wrong here @mdholloway : the spec test requires `section_id` and `type` to be present, but the definition additionally requires the `original` field. These need to match.

**Pull request merged**